dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/CompleteTests.cs

```diff
@@ -148,7 +148,7 @@ public async Task Get_FirstTimeSignInForEmailWithoutTrnUsingDefaultClient_Render
     {
         {
 #pragma warning disable CS0618 // Type or member is obsolete
-            CustomScopes.Trn,
+            CustomScopes.DqtRead,
```

Comment:
Yep I missed that (tbh I thought `dotnet format` would have tidied those up but it seems not).

Reply:
We probably should have made this `TrnRequirementType` thing configurable via the authorization endpoint after all; it would have saved us having to make more clients to test out all these different scenarios.